src/main/java/it/gov/pagopa/pu/processecexutions/dto/ArchivingExportFileRequestDTO.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.pu.processecexutions.dto;
+
+import it.gov.pagopa.pu.processecexutions.model.exportfile.ArchivingExportFileFilter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@NoArgsConstructor
+@SuperBuilder(toBuilder = true)
+public class ArchivingExportFileRequestDTO extends ExportFileRequestDTO<ArchivingExportFileFilter>{
```

Comment:
maybe we could move all exportFile to a dedicate package under dto?
```suggestion
public class ReceiptsArchivingExportFileRequestDTO extends ExportFileRequestDTO<ArchivingExportFileFilter>{
```